app/dashboard/models.py

```diff
@@ -84,6 +84,7 @@ class Bounty(SuperModel):
     title = models.CharField(max_length=255)
     web3_created = models.DateTimeField(db_index=True)
     value_in_token = models.DecimalField(default=1, decimal_places=2, max_digits=50)
+    eth_value_in_usdt = models.DecimalField(default=1, decimal_places=2, max_digits=50)
```

Comment:
Do we need this field? It doesn't appear that we're storing this on the model, but accessing the property defined below.

Reply:
Ah, great point. I was originally holding onto it but figured it would be best to get the most up-to-date value available. Fixing now.